storm-core/src/ui/public/flux.html

```diff
@@ -0,0 +1,139 @@
+<html>
```

Comment:
If we can have icon then we can place it to the left side of the upper right magnifier icon. If we can't, Text link `[Flux Viewer]` or so may be fine given that we can replace it anytime if we find better alternatives.